_data-prepper/pipelines/configuration/processors/anomaly-detector.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: default
+title: anomaly_detector
```

Comment:
I agree that this format is better in terms of reader comprehension.

Reply:
@Naarcha-AWS @carolxob Please work together to use @vagimeli feedback to decide on a consistent approach to this issue. Once you have decided on an approach, please apply it across the affected docs.

Reply:
@natebower @vagimeli @Naarcha-AWS Because this doc already appears in the "Processors" section, it would be redundant for now. My suggestion is to keep the naming convention consistent with the rest of the docset, then put in an issue (which I will own) to address that specific change in bulk. That way the entire docset naming convention will remain consistent.

Reply:
The issue that will address this is #3802.